scripts/plot_noise.py

```diff
@@ -78,30 +111,49 @@ def plot(savefig=False, **kwargs):
     plt.subplots_adjust(top=0.85)
     if savefig:
         plt.savefig("galactic_noise.png", bbox_inches="tight")
-    plt.show()
+    #plt.show()
 
-if __name__ == "__main__":
-    import argparse
-
-    parser = argparse.ArgumentParser(
-        description="Parser to select which noise quantity to plot. \
-        To Run: ./plot_noise.py \
-        Add --lst <int> for galactic noise. i.e ./plot_noise.py --lst 18 --savefig."
-        )
-    parser.add_argument(
-        "--lst",
-        default=18.0,
-        type=float,
-        help="lst for Local Sideral Time, galactic noise is variable with LST and maximal for 18h.",
-        )
-    parser.add_argument(
-        "--savefig",
-        action="store_true",
-        default=False,
-        help="don't add galactic noise.",
-    )
+def main():
+    parser = argparse.ArgumentParser(description="Plot function with command line arguments")
+    parser.add_argument('--savefig', action='store_true', help="Flag to save the figure")
```

Comment:
from what i can tell by reading the code, the savefig option might not be working. I dont see it being used.

Reply:
Yes there was a problem, i fixed it, now the savefig option works for all antenna models.